HW_1/Ex_2/Ex_2.py

```diff
@@ -0,0 +1,17 @@
+# # Задача 2: Найдите сумму цифр трехзначного числа
+
+# Пример
+# 100 -> 1 (1 + 0 + 0)
+
+
+# ___________________DECISION____________________
+
+num = int(input("Insert integer 3-digit number: "))
+
+if abs(num) < 100:
+    print("Inserted number is't 3-digit")
+else:
+    res = num % 10 + num // 10 % 10 + num // 100 % 10
```

Comment:
В последнем случае получать остаток от деления на 10 не обязательно.
```suggestion
    res = num % 10 + num // 10 % 10 + num // 100
```